parachain-template/runtime/src/lib.rs

```diff
@@ -113,6 +113,13 @@ pub type UncheckedExtrinsic =
 /// Extrinsic type that has already been checked.
 pub type CheckedExtrinsic = generic::CheckedExtrinsic<AccountId, RuntimeCall, SignedExtra>;
 
+/// Migrations to apply on runtime upgrade.
+pub type Migrations = (
+	cumulus_pallet_parachain_system::migration::Migration<Runtime>,
+	cumulus_pallet_xcmp_queue::migration::Migration<Runtime>,
+	cumulus_pallet_dmp_queue::migration::Migration<Runtime>,
+);
+
```

Comment:
```suggestion
```

No need for the template.

Reply:
Don't you want to have that in place so that everything is in place when you have some migration logic to run?